src/util.spec.ts

```diff
@@ -3,10 +3,18 @@ import { expect } from 'chai';
 import * as fsExtra from 'fs-extra';
 import { tempDir } from './testUtils.spec';
 import * as path from 'path';
+import * as dns from 'dns';
+import { createSandbox } from 'sinon';
+const sinon = createSandbox();
 
 describe('util', () => {
     beforeEach(() => {
         fsExtra.emptyDirSync(tempDir);
+        sinon.restore();
```

Comment:
We want to call restore in both before and after?

Reply:
I'm sure one or the other is _probably_ fine? It's a habit I've started. You never know if the last test cleaned itself up properly, so since it's just a line or two, I just do it both places.